doc/multi-o.md

```diff
@@ -0,0 +1,117 @@
+# Multi Orchestrator Setup
+
+This document describes the various ways an orchestrator could run multiple nodes and separate concerns. 
```

Comment:
I think it could be worth clarifying that the nodes would still accept payments on behalf of the same on-chain registered address while each node uses its own address.

Reply:
fixed in 4131a02b